definition_of_done.md

```diff
@@ -0,0 +1,37 @@
+# Definition of done
+
+This document clarifies, when a product backlog item is done. It will act as a checklist during the scrum development process.
+The individual points are a basis for acceptance criteria and are shortened or appended by specific measures as needed.
+
+## Documentation
+* [ ] Public api function descriptions (sphinx/doxygen are [autogenerated online](http://docs.ros.org/melodic/api/pilz_robots/html/))
+* [ ] API diagrams
+* [ ] [Tutorials](https://wiki.ros.org/pilz_robots/Tutorials/)
+* [ ] Overview on [ROS wiki](https://wiki.ros.org/pilz_robots)
+* [ ] Package Readme ([example pilz_robots](https://github.com/PilzDE/pilz_robots/blob/melodic-devel/README.md))
+* [ ] Good commit messages ([some tips](https://dev.to/jacobherrington/how-to-write-useful-commit-messages-my-commit-message-template-20n9
+))
+* [ ] Changelog.md updated
+* [ ] Copyright headers
+
+## Reviews
+* [ ] Soft- and hardware architecture
+* [ ] Code (coding rules, style guide)
+* [ ] Test review (test plan and individual test cases)
```

Comment:
Is it possible, that we first write tests and review them, before we start to implement?